pulpcore/cli/core/group.py

```diff
@@ -38,3 +40,38 @@ def create(pulp_ctx: PulpContext, entity_ctx: PulpEntityContext, name: str) -> N
 
 
 group.add_command(destroy_by_name)
+
+
+@group.group()
+@click.option("--groupname", required=True)
+@pass_entity_context
+@pass_pulp_context
+@click.pass_context
+def user(
+    ctx: click.Context, pulp_ctx: PulpContext, group_ctx: PulpGroupContext, groupname: str
+) -> None:
+    ctx.obj = PulpGroupUserContext(pulp_ctx)
+    ctx.obj.group = group_ctx.find(name=groupname)
+
+
+user.add_command(list_entities)
+
+
+@user.command()
+@click.option("--username", required=True)
+@pass_entity_context
+def add(entity_ctx: PulpGroupUserContext, username: str) -> None:
+    # This href_key seems to be a bug in the apidocs: it should be "auth_group_href"
```

Comment:
Was this fixed as part of https://github.com/pulp/pulpcore/pull/1066 ?

Reply:
Yes, do we need the version specific workaround, or do we just wait for this to merge?